src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -80,7 +81,13 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
   private def addProps(depositPropertiesFactory: DepositPropertiesFactory, maybeOutputDir: Option[File])
                       (bagParentDir: File): Try[Boolean] = {
     logger.debug(s"creating application.properties for $bagParentDir")
-    val changedMetadata = Seq("bag-info.txt", "metadata/amd.xml", "metadata/dataset.xml", "metadata/provenance.xml").map(Paths.get(_))
+    val changedMetadata = Seq(
+      "bag-info.txt",
+      "metadata/amd.xml", "metadata/emd.xml", // in practice one of both
```

Comment:
```suggestion
      "metadata/amd.xml", 
      "metadata/emd.xml", 
```
is this is leftover of the amd/emd confusion?

Reply:
indeed